### PR TITLE
Cds 49945 servicev1 eol notice

### PR DESCRIPTION
**Pull request by michaelcretzman**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [x ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* https://harness.atlassian.net/browse/CDS-49945

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by thisrohangupta — APPROVED** [review]
This looks good and its what @michaelcretzman and I discussed through slack.
Content looks good, the YAML snippets are in their to help educate the user what parts of the object has changed.

File: docs/continuous-delivery/onboard-cd/upgrading/upgrade-cd-v2.md [hunk 1]
```diff
@@ -18,64 +18,993 @@ Except for specific Harness accounts, the current v1 experience will be removed
 
 ## Why make the change?
 
-Service and environment v1 tied the service and environment entities to the pipelines where they were defined. This prevented you from making a change to a service or environment and having it updated in multiple pipelines at once.
+The v2 experience has more robust service and environment entities. V2 has service variables, independent infrastructure definitions, environment groups, and file and variable overrides. 
 
-The new v2 experience is designed to provided users and organizations with a simpler configuration and the ability to scale deployments using services and environments.
+With v2, you'll notice an major reduction in the configuration included into pipelines. These changes are also reflected in the the Harness APIs.
```
Comment (douglas-j-bothwell) [inline docs/continuous-delivery/onboard-cd/upgrading/upgrade-cd-v2.md:23]:
```suggestion
With v2, you'll notice an major reduction in the configuration included in pipelines. These changes are also reflected in the the Harness APIs.
```

Comment (douglas-j-bothwell) [reply docs/continuous-delivery/onboard-cd/upgrading/upgrade-cd-v2.md:23]: "With v2, you'll notice an major reduction in the configuration included in pipelines." is a little fuzzy. Perhaps something like "V2 requires much less YAML code to define pipelines."

File: docs/continuous-delivery/onboard-cd/upgrading/upgrade-cd-v2.md [hunk 2]
```diff
@@ -18,64 +18,993 @@ Except for specific Harness accounts, the current v1 experience will be removed
...
-
-### Existing Harness users and projects
-
-There is no impact on your existing pipelines using v1 services and environments. 
-
-After 1/31/2023, when you create a new stage in an existing pipeline, the v2 experience will be the default. 
+Here's a summary of the impact to Harness accounts:
 
-### New Harness users and projects
+- The changes only impact your CD module. No other modules are impacted.
```
Comment (douglas-j-bothwell) [inline docs/continuous-delivery/onboard-cd/upgrading/upgrade-cd-v2.md:33]:
```suggestion
- The changes impact your CD module only. No other modules are impacted.
```

File: docs/continuous-delivery/onboard-cd/upgrading/upgrade-cd-v2.md [hunk 3]
```diff
@@ -18,64 +18,993 @@ Except for specific Harness accounts, the current v1 experience will be removed
...
-- Terraform Provider:
-  - [harness_platform_service (Resource)](https://registry.terraform.io/providers/harness/harness/latest/docs/resources/platform_service)
-  - [harness_platform_environment (Resource)](https://registry.terraform.io/providers/harness/harness/latest/docs/resources/platform_environment)
-  - [harness_platform_infrastructure (Resource)](https://registry.terraform.io/providers/harness/harness/latest/docs/resources/platform_infrastructure)
+Please review these important dates before upgrading.
 
+- Service and environment v1 APIs are being deprecated in March 2023.
+- On the official EOL date (Jan 31, 2023), v1 services and environments will continue to work as before. After March 2023, pipelines with v1 services and environments might experience issues or even failures because Harness will not update v1 with fixes or enhancements after that date.
+- Harness will globally enable service and environments v2 APIs for all customers at the end of January 2023.
+- The forced change is needed to reduce the migration effort needed for users.
```
Comment (douglas-j-bothwell) [inline docs/continuous-delivery/onboard-cd/upgrading/upgrade-cd-v2.md:50]:
```suggestion
- The forced change will reduce the migration effort needed for users.
```

File: docs/continuous-delivery/onboard-cd/upgrading/upgrade-cd-v2.md [hunk 4]
```diff
@@ -18,64 +18,993 @@ Except for specific Harness accounts, the current v1 experience will be removed
...
+- Harness has an automated tool to help migrate your services and environments from v1 to v2.
+
+We are happy to assist you in your migration to the new v2 experience. For any questions or comments, please contact `support@harness.io` or your CSM. Together we can work through discovery, migration planning, and assist with the process.
+
+## API changes
+
+Harness has introduced the follow new APIs to support the v2 experience:
```
Comment (douglas-j-bothwell) [inline docs/continuous-delivery/onboard-cd/upgrading/upgrade-cd-v2.md:57]:
```suggestion
Harness has introduced the following new APIs to support the v2 experience:
```

File: docs/continuous-delivery/onboard-cd/upgrading/upgrade-cd-v2.md [hunk 5]
```diff
@@ -18,64 +18,993 @@ Except for specific Harness accounts, the current v1 experience will be removed
...
+- For details on service v2, go to [Services and environments overview](../cd-concepts/services-and-environments-overview.md).
+- The service entity is now moved from the pipeline to a standalone entity. The service contains the following components:
+  - **Name**, **Description**, **Tag**, **Id**. These are the same as in the service v1 experience.
+  - Manifests and artifacts. The service manifests and artifacts are now mapped in the service. They are moved out of the pipeline **Service** tab.
+  - Service variables. Service variables are now associated with the Service and can be overridden at the environment level.
+
+
+#### Service YAML updates
+
+Here is a a YAML sample the demonstrates the changes in v2.
```
Comment (douglas-j-bothwell) [inline docs/continuous-delivery/onboard-cd/upgrading/upgrade-cd-v2.md:95]:
```suggestion
Here is a a YAML sample that demonstrates the changes in v2.
```

File: docs/continuous-delivery/onboard-cd/upgrading/upgrade-cd-v2.md [hunk 6]
```diff
@@ -18,64 +18,993 @@ Except for specific Harness accounts, the current v1 experience will be removed
...
+
+- Environments are now standalone objects with global environment variables and manifest configurations.
+- V2 **Environment Service Overrides** and **Environment Variables** can override service variables when the service is deployed into a given environment. Based on the variable name, Harness can override the service variable with the environment variable value.
+- Harness has introduced service-specific environment overrides where users can define specific services and variables they want to override for a given environment.
+- For details on environments v2, go to [Services and environments overview](../cd-concepts/services-and-environments-overview).
+
+
+#### Environment groups
+
+V2 introduces environment groups. You can now group environments and manage them at scale.
```
Comment (douglas-j-bothwell) [inline docs/continuous-delivery/onboard-cd/upgrading/upgrade-cd-v2.md:307]:
```suggestion
V2 introduces environment groups. You can now aggregate environments and manage them at scale. 
```

File: docs/continuous-delivery/onboard-cd/upgrading/upgrade-cd-v2.md [hunk 7]
```diff
@@ -18,64 +18,993 @@ Except for specific Harness accounts, the current v1 experience will be removed
...
+```
+
+### Infrastructure definition
+
+Harness has taken the infrastructure definition that was originally defined in the pipeline and moved it to the environment. For more information, go to [ Services and environments overview](../cd-concepts/services-and-environments-overview).
+
+Here are the changes to infrastructure definition:
+ 
+- The infrastructure definition can only be associated with 1 environment.
+- The infrastructure definition is required for a pipeline execution to run.
```
Comment (douglas-j-bothwell) [inline docs/continuous-delivery/onboard-cd/upgrading/upgrade-cd-v2.md:473]:
```suggestion
- The infrastructure definition is required to run a pipeline execution. 
```

**Review by douglas-j-bothwell — DISMISSED** [review]

**Review by thisrohangupta — APPROVED** [review]
approving this with the assumption that you’ll consider my suggestions

**Pull request merged**